docs/ui/sherpa_astro_ui.rst

```diff
@@ -135,6 +135,7 @@ burden...
       get_model_component
       get_model_component_image
       get_model_component_plot
+      get_model_components_plot
```

Comment:
Is this part of this PR?

Reply:
This is mentioned in the second commit message, but I forgot to add anything to the overall commit message:

> This also adds a number of routines from the 4.16.0 and 4.16.1
releases which I forgot to add.

Reply:
I've added a note to the "commit message" to explain this.